create authorization; i firebase

SerhiiPochtovyk opened the pull request:
my first commit

SerhiiPochtovyk committed:
create authorization; i firebase

AsphaltHedgehog reviewed (changes requested):
Я так понял там ничего не доделано ещё, я больше приверженец того чтобы сначала всё закончить, будут вопросы пиши в дискорд)

SerhiiPochtovyk committed:
create auth; install firebase;

SerhiiPochtovyk committed:
Authorization_

abracadabra

SerhiiPochtovyk committed:
Update authorise.html

SerhiiPochtovyk committed:
create scrollUp btn;

SerhiiPochtovyk committed:
var3 for modal-form-login

реалізовано відкриття модального вікна для форми регістрації користувача;
створено html для форми авторизації;

SerhiiPochtovyk commented:
привіт. видали цей реквест, щоб не висіли паролі до firebase. Дякую!

AsphaltHedgehog commented:
> привіт. видали цей реквест, щоб не висіли паролі до firebase. Дякую!

Хз если честно как это сделать, попробуй просто удалить их и комитнуть, а как именно скрыть их хз, поменяй пароль
